HW_3/supportive.py

```diff
@@ -0,0 +1,30 @@
+def list_double_item(lst:list, tutle_list:bool = False, ignor_simvols:str ='') -> list:
+    lst.sort(key=str)
```

Comment:
Ничего не поменяется
```suggestion
    lst.sort()
```

Reply:
поменяеся, т.к. в словаре могут быть типы которые автоматически нельзя сравнить

Reply:
Да, вы правы. Если применять эту функцию к списку, содержащему нехэширемые объекты.